AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Support/Cancellable.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+//Note: There is already a Cancellable in Core/Support/Cancellable.swift, which does not seem to be
```

Comment:
Rather than re-implement, import the core cancellable with `import Amplify` where needed. Otherwise we'll have Combine.Cancellable, Amplify.Cancellable, and AWSDataStoreCategoryPlugin.Cancellable all potentially conflicting.

Reply:
I tried importing cancellable with import Amplify, and tried extending classes with Amplify.Cancellable, but wasn't able to fix this.  I opened an issue here to take a look at cleaning this up:
https://github.com/aws-amplify/amplify-ios/issues/328

Reply:
That's weird, by looking at the type declaration I don't see anything that could block it from being used outside of core when importing `Amplify`. Is it an Xcode cache issue?